Lib/copy.py

```diff
@@ -105,7 +105,7 @@ def copy(x):
           types.BuiltinFunctionType, types.EllipsisType,
           types.NotImplementedType, types.FunctionType, types.CodeType,
           weakref.ref, super})
-_copy_builtin_containers = frozenset({list, dict, set, bytearray})
+_copy_builtin_containers = frozenset({list, dict, frozendict, set, bytearray})
```

Comment:
```suggestion
_copy_builtin_containers = frozenset({list, dict, set, bytearray})
```
The frozendict can be added to the `_copy_atomic_types` instead.

Reply:
Good idea! Done. I also added tests.